tests/graze_composer_env.bats

```diff
@@ -51,136 +51,130 @@ teardown() {
-  echo 'status:' $status
-  echo 'output:' $output
+  echo "status: $status"
+  echo "output: $output"
   [ "$status" -eq 0 ]
   [ "$output" = "https://github.com/graze/docker-composer" ]
 }
 
 @test "the image has a vcs-ref label set to the current head commit in github" {
   run bash -c "docker inspect graze/composer:$tag | jq -r '.[].Config.Labels.\"org.label-schema.vcs-ref\"'"
-  echo 'status:' $status
-  echo 'output:' $output
+  echo "status: $status"
+  echo "output: $output"
   [ "$status" -eq 0 ]
   [ "$output" = `git rev-parse --short HEAD` ]
 }
 
 @test "the image has a build-date label" {
   run bash -c "docker inspect graze/composer:$tag | jq -r '.[].Config.Labels.\"org.label-schema.build-date\"'"
-  echo 'status:' $status
-  echo 'output:' $output
+  echo "status: $status"
+  echo "output: $output"
   [ "$status" -eq 0 ]
   [ "$output" != "null" ]
 }
 
 @test "the image has a vendor label" {
   run bash -c "docker inspect graze/composer:$tag | jq -r '.[].Config.Labels.\"org.label-schema.vendor\"'"
-  echo 'status:' $status
-  echo 'output:' $output
+  echo "status: $status"
+  echo "output: $output"
   [ "$status" -eq 0 ]
   [ "$output" = "graze" ]
 }
 
 @test "the image has a name label" {
   run bash -c "docker inspect graze/composer:$tag | jq -r '.[].Config.Labels.\"org.label-schema.name\"'"
-  echo 'status:' $status
-  echo 'output:' $output
+  echo "status: $status"
+  echo "output: $output"
   [ "$status" -eq 0 ]
   [ "$output" = "composer" ]
 }
 
 @test "the image has a version label" {
   run bash -c "docker inspect graze/composer:$tag | jq -r '.[].Config.Labels.\"org.label-schema.version\"'"
-  echo 'status:' $status
-  echo 'output:' $output
+  echo "status: $status"
+  echo "output: $output"
   [ "$status" -eq 0 ]
   [ "$output" = "${COMPOSER_VER}-php${PHP_VER}" ]
 }
 
 @test "the image has a docker.cmd label" {
   run bash -c "docker inspect graze/composer:$tag | jq -r '.[].Config.Labels.\"org.label-schema.docker.cmd\"'"
-  echo 'status:' $status
-  echo 'output:' $output
+  echo "status: $status"
+  echo "output: $output"
   [ "$status" -eq 0 ]
   [ "$output" = "docker run --rm -it -v \$(pwd):/usr/src/app -v ~/.composer:/home/composer/.composer -v ~/.ssh/id_rsa:/home/composer/.ssh/id_rsa:ro graze/composer:${COMPOSER_VER}-php${PHP_VER}" ]
 }
 
-@test "the composer wrapper has been copied" {
-  run docker run --rm --entrypoint=/bin/sh graze/composer:$tag -c '[ -x /usr/local/bin/composer-wrapper ]'
-  echo 'status:' $status
-  [ "$status" -eq 0 ]
-}
-
-@test "the image entrypoint should be the composer wrapper" {
+@test "the image entrypoint should be composer" {
   run bash -c "docker inspect graze/composer:$tag | jq -r '.[].Config.Entrypoint[]'"
-  echo 'status:' $status
-  echo 'output:' $output
+  echo 'status:' "$status"
+  echo 'output:' "$output"
   [ "$status" -eq 0 ]
-  [ "$output" = "/usr/local/bin/composer-wrapper" ]
+  [ "$output" = "/usr/bin/php/usr/local/bin/composer" ]
```

Comment:
This looks weird. I know it's a consequence of the fact that two lines are returned from `docker inspect` as we have talked about, but it looks odd to see the values munged together like this.